doc/ParkGuestItems.md

```diff
@@ -0,0 +1,15 @@
+Guests can buy, carry and use/consume several types of item. These are:
```

Comment:
move to user manual?

Should probably be cross-referenced with the actual items that exist in some item enum